CMakeLists.txt

```diff
@@ -15,6 +15,11 @@ endif()
 project(SwiftTesting
   LANGUAGES CXX Swift)
 
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  find_package(dispatch)
```

Comment:
If we go this route instead of using the build-directory configurations, we'll need to include a `Finddispatch.cmake` and `FindFoundation.cmake` in our modules directory and move these down below the `list(APPEND CMAKE_MODUKE_PATH ...` on line 25.

Reply:
I think `find_package()` tries Module mode first then falls backs to Config mode IIUC.
> When the [basic signature](https://cmake.org/cmake/help/latest/command/find_package.html#basic-signature) is used, the command searches in Module mode first. If the package is not found, the search falls back to Config mode.

Reply:
Sure, but I think we still would want the `Find<PackageName>.cmake` modules or I don't really see much point in not just using the `CONFIG` form.

Reply:
Let's just use `CONFIG` then.